Source/Rhetos.Extensibility/PluginScanner.cs

```diff
@@ -67,8 +67,11 @@ public IEnumerable<PluginInfo> FindPlugins(Type pluginInterface)
         {
             var assemblies = ListAssemblies(findAssemblies);
             MultiDictionary<string, PluginInfo> plugins = null;
+            var namesToPaths = assemblies.ToDictionary(path => Path.GetFileName(path), path => path, StringComparer.InvariantCultureIgnoreCase);

```

Comment:
ToDictionary fails with duplicate key when using (localized NuGet packages](https://docs.microsoft.com/en-us/nuget/create-packages/creating-localized-packages). For example, "Rhetos.LightDMS" references a localized package.